clang-tools-extra/clang-tidy/readability/RedundantParenthesesCheck.cpp

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
```suggestion
```

Is code generated by Python script? Will  be good idea to fix leading newline there.

Reply:
fix script in #160019